### PR TITLE
Clean up bug

### LINKED ISSUE #571: BUG: Clean Up Agents Button
**Bug Description**

I just hit the clear agents button for our team, and only some of the unhealthy agents were cleared.

**Steps To Reproduce**

1. Went to Agents tab
2. Pressed "Clean Up" button
3. Only some agents cleared
4. Pressed button again
5. Agents remained

**Browsers Tested:**
* [X] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Edge
* [ ] IE

----------
**Notes**

https://app.logrocket.com/qetfiw/cloud-web-ui/s/4-2368e0da-c3e5-46da-b59f-e16dbcc65015/0?filters=%255B%257B%2522id%2522%253A%2522HJOs2-6yd%2522%252C%2522type%2522%253A%2522userID%2522%252C%2522operator%2522%253A%257B%2522name%2522%253A%2522is%2522%252C%2522type%2522%253A%2522IS%2522%252C%2522hasStrings%2522%253Atrue%252C%2522autocompleteEnabled%2522%253Atrue%257D%252C%2522strings%2522%253A%255B%2522auth0%257C5d267468a614960c643dab0e%2522%255D%257D%255D&fromTab=latest
Screen Shot 2021-01-25 at 10 30 36 PM

### PR DESCRIPTION
**Pull request by zhen0**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [x] add/update tests (or don't, for reasons explained below)

## Describe this PR
- Closes #571 
- I can't actually reproduce this bug reported in #571 but I can see it on the log rocket link. I've added some better error handling so we can track it and have more information to fix it if we see it again in the future.

### CODE REVIEW

File: src/components/Agents/Agents.vue [hunk 1]
```diff
@@ -106,31 +108,53 @@ export default {
...
+            }, 10000)
+          })
+          .catch(e => {
+            LogRocket.captureError(e)
+            setTimeout(() => {
+              agent.isDeleting = false
+              this.setAlert({
+                alertShow: true,
+                alertMessage: 'Error clearing agents',
+                alertType: 'error'
```
Comment (znicholasbrown) [inline src/components/Agents/Agents.vue:147]: Won't this attempt to show an alert for every agent? That could get messy in some scenarios. Maybe add a counter for failed mutations that we can show in a single alert at the end? (and maybe can be included in the logrocket error)

File: src/components/Agents/Agents.vue [hunk 2]
```diff
@@ -243,7 +267,13 @@ export default {
             <v-btn text tile @click="cleanUpDialog = false">
               Cancel
             </v-btn>
-            <v-btn dark color="red" depressed @click="clearUnhealthyAgents">
+            <v-btn
+              :loading="clearingAgents"
```
Comment (znicholasbrown) [inline src/components/Agents/Agents.vue:271]: 👍

**Review by znicholasbrown — DISMISSED** [review]
LGTM just 1 change

File: src/components/Agents/Agents.vue (hunk 1 above)
Comment (zhen0) [reply src/components/Agents/Agents.vue:147]: Good point - can update that

**Review by znicholasbrown — APPROVED** [review]
LGTM @zhen0 !

**Pull request merged**